TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss

```diff
@@ -0,0 +1,99 @@
+@use '@/variables.module.scss' as v;
+
+.tf-button {
+    background: v.$dark-grey-gradient;
+    box-shadow: 0px 2px 10px #00000099;
+    border: 1px solid #ffffff1a;
+    border-radius: 10px;
+
+    user-select: none;
+
+    &--hovered {
+        box-shadow: 0px 0px 30px v.$ultraleap-green;
+    }
+
+    &--pressed {
+        box-shadow: 0px 0px 30px white;
+        transform: scale(0.95, 0.95);
+    }
+
+    &--horizontal {
+        display: flex;
+        flex-direction: row;
+        height: 200px;
+
+        .tf-button-text-container {
+            width: 65%;
+            align-self: flex-end;
+            padding-bottom: 3%;
+        }
+
+        img {
+            margin: auto;
+            width: 16%;
+        }
+    }
+
+    &--text {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        p {
+            padding: 0;
+            text-align: center;
+            line-height: 40px;
+        }
+    }
+
+    &--misc {
+        width: 25vw;
+        height: 4rem;
+        @media (orientation: landscape) {
+            width: 12vw;
+        }
+
+        padding: 0.3rem;
+
+        background-image: linear-gradient(180deg, #353535, #2d2d2d), linear-gradient(180deg, #00eb86, #00cdcf);
+        background-clip: content-box, border-box;
+        border-radius: 50px;
+
+        > h1 {
+            font-size: 1.5rem !important;
```

Comment:
`!important` is required here because it needs to overwrite the base class' `h1` styling. Could have a look at more specific classes, but would take a bit more time

Reply:
Moving the h1{} block in .tf-button above the override h1 stlying meaning you don't need important as styles lower down in the css file overwrite those above (at the moment the h1{} block is below the --misc { >h1} so the default styling gets priority)

Reply:
Think there are some other cases of !important in that file which can be solved in the same way (keen on avoiding !important unless it's overwriting a third party stylesheet and is unavoidable)

Reply:
All removed!